sdk/nodejs/runtime/stack.ts

```diff
@@ -25,16 +26,6 @@ import { isDryRun, isQueryMode, setRootResource } from "./settings";
  */
 export const rootPulumiStackTypeName = "pulumi:pulumi:Stack";
 
-// The stack resource needs to be a a true global, so that if we end up with multiple Pulumi modules loaded
-// they all resolve to the same variable.
-declare global {
-    var stackResource: Stack | undefined; // eslint-disable-line no-var
-}
-
-// Get the root stack resource for the current stack deployment
-export function getStackResource(): Stack | undefined {
-    return globalThis.stackResource;
-}
```

Comment:
Since this is exported, for better or worse, it's part of the public API. I think we need to continue exporting a `getStackResource` from this module. Perhaps it just returns the result of `getStackResource` from `./state` (although that import will need to be aliased if this is kept here).